acrobat/blocks/app-banner/app-banner.css

```diff
@@ -0,0 +1,167 @@
+.placeholder-banner {
+  position: relative;
+  top: 0;
+  left: 0;
+  height: 76px;
+}
+
+.has-app-banner header.global-navigation {
+  top: 76px;
+}
+
+.app-banner-close {
+  display: inline;
+  width: 24px;
+  padding: 30px 0;
+  text-align: center;
+  color: grey;
+  font-size: 20px;
+  position: relative;
+  top: 43%;
+  transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  z-index: 2;
+  box-sizing: content-box;
+}
+
+.app-banner-icon {
+  border-radius: 10px;
+  width: 63px;
+  height: 63px;
+  margin-right: 10px;
+  padding: 10px 0;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  z-index: 2;
+  box-sizing: content-box;
+}
+
+.app-banner-icon img {
+  width: 63px;
+  height: 63px;
+  border-radius: 10px;
+  box-sizing: content-box;
+}
+
+.app-banner-title {
+  min-height: 2px;
+  font-weight: bold;
+  color: #333333;
+  padding-top: 5px;
+  padding-bottom: 1px;
+  font-size: 12px;
+  height: 14px;
+  box-sizing: content-box;
+}
+
+.app-banner-description {
+  min-height: 2px;
+  overflow: hidden;
+  color: #333333;
+  font-size: 11px;
+  height: 18px;
+  box-sizing: content-box;
+}
+
+.app-banner-button {
+  font-size: 12px;
+  border: 1px solid;
+  border-radius: 10px;
+  padding: 10px 10px;
+  text-decoration: none;
+  color: #1CADCE;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 1.15px;
+  display: inline;
+  vertical-align: middle;
+  margin-right: 9px;
+  position: relative;
+  z-index: 2;
+  height: 14px;
+  box-sizing: content-box;
+}
+
+.app-banner-stars {
+  display: inline-block;
+  color: #ffb400;
+  letter-spacing: -2px;
+  font-size: 16px;
+  margin-left: -2px;
+  box-sizing: content-box;
+}
+
+.app-banner-reviews {
+  min-height: 10px;
+  min-width: 10px;
+  font-size: 12px;
+  color: #777;
+  display: inline-block;
+  margin-left: 4px;
+  position: relative;
+  top: -1px;
+  box-sizing: content-box;
+}
+
+.app-banner-stars span {
+  position: relative;
+  box-sizing: content-box;
+}
+
+.app-banner-content {
+  width: 100%;
+  overflow: hidden;
+  height: 76px;
+  background: white;
+  color: #333;
+  border-top: 1px solid #ddd;
+  display: flex;
+  box-sizing: content-box;
```

Comment:
You can remove all `box-sizing: content-box;` because this is default value for box-sizing https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
@zagi25 This is required to forcibly set to default on pages where the box-sizing is set to border-box by preceding blocks in Milo/Gnav and its interfering with app banner elements.